packages/components/pie-webc/src/componentService.js

```diff
@@ -1,3 +1,4 @@
+import path from 'path';
```

Comment:
I don't think we need this import, we can use `this.path` instead

Reply:
Good point